components/console_cmd_ping/Kconfig.projbuild

```diff
@@ -0,0 +1,21 @@
+menu "Ping command Configuration"
+
+    config PING_CMD_AUTO_REGISTRATION
```

Comment:
to simplify this, maybe we can support only one auto-registration config per component?

Reply:
Done